src/styles/header.scss

```diff
@@ -0,0 +1,99 @@
+.header {
+    position: fixed;
+    top: 0;
+    left: 0;
+  
+    display: flex;
+    align-items: center;
+    justify-content: start;
+    flex-direction: column;
+  
+    height: 100%;
+    width: 160px;
+    padding-top: 20px;
+  
+    background-color: rgb(48, 48, 48);
+  
+    overflow-x: hidden;
+    z-index: 1;
+  }
+  
+  .container {
+    position: relative;
+    display: block;
+    margin: 10px 0;
+    transition: all .3s;
+    cursor: pointer;
+  
+    &:not(.triangle, .dotted-line) {
+      background: #ffffff;
+    }
+  
+    &:hover:not(.triangle, .dotted-line) {
+      background: peachpuff;
```

Comment:
I'm OK with it, but usually, we don't use color names for definitions. HEX or RGB

Reply:
Changed color names to HEX